pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -41,24 +36,35 @@ In the PWA Studio project's root directory, run the following command to install
 yarn install
 ```
 
-## Step 3. Specify the Magento backend server
+## Step 3. Build the pwa-buildpack package
+
+Setting up Venia requires the use of the `buildpack` CLI tool in the pwa-buildpack package.
+
+Build the pwa-buildpack package to enable the use of this tool:
+
+```sh
+yarn workspace @magento/pwa-buildpack build
+```
+
+## Step 4. Create the `.env` file
 
-In the root directory, run `yarn buildpack create-env-file packages/venia-concept`.
+In the root directory, use the `create-env-file` subcommand for the `buildpack` CLI tool to create a `.env` file for Venia.
 
-In the newly generated `packages/venia-concept/.env` file set the value of `MAGENTO_BACKEND_URL` to the URL of a running Magento instance.
+This command generates a `packages/venia-concept/.env` file where you can set the value of `MAGENTO_BACKEND_URL` to the URL of a running Magento instance.
 
 If you are contributing to Venia development or exploring its features, you can use the default `MAGENTO_BACKEND_URL` value.
 This URL points to a cloud instance of Magento 2.3.1 with the [Venia sample data][] installed.
 
-**Example:**
+**Example `.env` file entry:**
+
 ``` text
 MAGENTO_BACKEND_URL="https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/"
 ```
 
-You can do this all in a single command in `bash` or `sh`:
+You can also create the `.env` file and set the `MAGENTO_BACKEND_URL` value in a single `bash` or `sh` command:
 
 ```sh
-MAGENTO_BACKEND_URL=https://some.url yarn buildpack create-env-file packages/venia-concept
+MAGENTO_BACKEND_URL="https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" yarn buildpack create-env-file packages/venia-concept
```

Comment:
Was this move to an actual url intentional?

Reply:
yes, this makes it easier to just copy and paste the command from the docs

Reply:
yeah I wanted to streamline the process as much as possible for users